src/conways.rs

```diff
@@ -0,0 +1,234 @@
+use std::fmt::Display;
+
+#[derive(Clone, Copy, PartialEq, Debug)]
+enum CellState {
+    Alive,
+    Dead
+}
+
+pub struct Conways {
+    world: Vec<Vec<CellState>>,
```

Comment:
Related to the comment I did above, maybe would be better to rename this to `cells` (or `grid`, depending on how you named the struct).